src/main/java/Parser.java

```diff
@@ -0,0 +1,102 @@
+
+    public static void handleCommand(String command, ArrayList<Task> storedTasks) throws DukeException {
+        if (command.equals("bye")) {
+            System.out.println(HORIZONTAL_LINE + "\n  Bye. Hope to see you again soon!\n" + HORIZONTAL_LINE);
+        } else if (command.equals("list")) {
+            System.out.println(HORIZONTAL_LINE);
+            for (int i = 0; i < storedTasks.size(); i++) {
+                System.out.println("  " + String.valueOf(i + 1) + ". " + storedTasks.get(i));
+            }
+            System.out.println(HORIZONTAL_LINE);
+        } else if (command.split(" ").length == 2 && command.split(" ")[0].equals("mark")) {
+            int taskNumber = Integer.parseInt(command.split(" ")[1]) - 1;
+            storedTasks.get(taskNumber).markAsDone();
+            System.out.println(HORIZONTAL_LINE + "\n  Nice! I've marked this task as done:\n"
+                    + storedTasks.get(taskNumber) + "\n" + HORIZONTAL_LINE);
+        } else if (command.split(" ").length == 2 && command.split(" ")[0].equals("unmark")) {
+            int taskNumber = Integer.parseInt(command.split(" ")[1]) - 1;
+            storedTasks.get(taskNumber).markAsUndone();
+            System.out.println(HORIZONTAL_LINE + "\n  OK, I've marked this task as not done yet:\n"
+                    + storedTasks.get(taskNumber) + "\n" + HORIZONTAL_LINE);
+        } else if (command.split(" ").length == 2 && command.split(" ")[0].equals("delete")) {
+            int taskNumber = Integer.parseInt(command.split(" ")[1]) - 1;
+            System.out.println(HORIZONTAL_LINE + "\n  Noted. I've removed this task:\n    "
+                    + storedTasks.get(taskNumber));
+            storedTasks.remove(taskNumber);
+            System.out.println("  Now you have "
+                    + String.valueOf(storedTasks.size()) + " tasks in the list.\n" + HORIZONTAL_LINE);
+        } else if (command.split(" ").length > 1 && command.split(" ")[0].equals("todo")) {
+            ArrayList<String> commandDelimited = new ArrayList<String>(Arrays.asList(command.split(" ")));
+            if (commandDelimited.size() == 1) {
+                throw new DukeException("The description of a todo cannot be empty.");
+            }
+            StringBuilder description = new StringBuilder();
+            for (int i = 1; i < commandDelimited.size(); i++) {
+                description.append(commandDelimited.get(i));
+                if (i != commandDelimited.size() - 1) {
+                    description.append(" ");
+                }
+            }
+            Task todo = new Todos(description.toString());
+            storedTasks.add(todo);
+            System.out.println(
+                    HORIZONTAL_LINE + "\n  Got it. I've added this task:\n  " + todo + "\n  Now you have "
+                            + String.valueOf(storedTasks.size()) + " tasks in the list.\n" + HORIZONTAL_LINE);
```

Comment:
Should this be extracted to the ToDo class or TaskList class instead? I noticed this for the deadlines and event class too.